frb/builds/build_frbs.py

```diff
@@ -85,8 +85,8 @@ def frb_180924():
     frb180924.lpol = 80.  # %
     frb180924.lpol_err = 10.
     # Error ellipse
-    frb180924.set_ee(a=100./1e3, b=100./1e3, theta=0., cl=68.) #statistical
-    frb180924.set_ee(a=0.09, b=0.09, theta=0, cl=68., stat=False) #systematic
+    frb180924.set_ee(a=0.06, b=0.07, theta=0., cl=68.) # Statistical (Day+2020)
+    frb180924.set_ee(a=0.07, b=0.09, theta=0., cl=68.,stat=False) # Systematic
```

Comment:
`a` can't be less than `b` for an ellipse.
I suspect you'll need to swap these.  
Keep in mind how `theta` is defined.

Reply:
Setting PA=90.